diff --git a/node_modules/expo-text-extractor/android/build.gradle b/node_modules/expo-text-extractor/android/build.gradle
index 7f1375c..912c9ea 100644
--- a/node_modules/expo-text-extractor/android/build.gradle
+++ b/node_modules/expo-text-extractor/android/build.gradle
@@ -41,6 +41,6 @@ android {
     abortOnError false
   }
   dependencies {
-    implementation 'com.google.android.gms:play-services-mlkit-text-recognition:19.0.1'
+    implementation 'com.google.android.gms:play-services-mlkit-text-recognition-korean:16.0.1'
   }
 }
diff --git a/node_modules/expo-text-extractor/android/src/main/java/expo/modules/textextractor/ExpoTextExtractorModule.kt b/node_modules/expo-text-extractor/android/src/main/java/expo/modules/textextractor/ExpoTextExtractorModule.kt
index 3ffdf31..404417a 100644
--- a/node_modules/expo-text-extractor/android/src/main/java/expo/modules/textextractor/ExpoTextExtractorModule.kt
+++ b/node_modules/expo-text-extractor/android/src/main/java/expo/modules/textextractor/ExpoTextExtractorModule.kt
@@ -3,7 +3,7 @@ package expo.modules.textextractor
 import android.net.Uri
 import com.google.mlkit.vision.common.InputImage
 import com.google.mlkit.vision.text.TextRecognition
-import com.google.mlkit.vision.text.latin.TextRecognizerOptions
+import com.google.mlkit.vision.text.korean.KoreanTextRecognizerOptions
 import expo.modules.kotlin.Promise
 import expo.modules.kotlin.exception.CodedException
 import expo.modules.kotlin.modules.Module
@@ -32,7 +32,7 @@ class ExpoTextExtractorModule : Module() {
         }
 
         val inputImage = InputImage.fromFilePath(context, uri)
-        val recognizer = TextRecognition.getClient(TextRecognizerOptions.DEFAULT_OPTIONS)
+        val recognizer = TextRecognition.getClient(KoreanTextRecognizerOptions.Builder().build())
 
         recognizer.process(inputImage)
           .addOnSuccessListener { visionText ->
diff --git a/node_modules/expo-text-extractor/ios/ExpoTextExtractorModule.swift b/node_modules/expo-text-extractor/ios/ExpoTextExtractorModule.swift
index 102a7a0..019890a 100644
--- a/node_modules/expo-text-extractor/ios/ExpoTextExtractorModule.swift
+++ b/node_modules/expo-text-extractor/ios/ExpoTextExtractorModule.swift
@@ -29,6 +29,7 @@ public class ExpoTextExtractorModule: Module {
 
                     promise.resolve(recognizedTexts)
                 }
+                request.recognitionLanguages = ["ko-KR", "en-US"]
 
                 try requestHandler.perform([request])
             } catch {
